server/src/entity/MenuItem.ts

```diff
@@ -121,6 +121,12 @@ export class MenuItem extends BaseEntity {
   })
   public isActive: boolean;
 
+  @OneToMany(() => Review, (review) => review.menuItem)
```

Comment:
Invalid syntax `@ManyToOne(type => User, user => user.photos)`

Reply:
What do you mean by invalid syntax... it works